packages/translations/src/languages/sv.ts

```diff
@@ -174,6 +174,17 @@ export const svTranslations: DefaultTranslationsObject = {
     toggleBlock: 'Växla block',
     uploadNewLabel: 'Ladda upp ny {{label}}',
   },
+  folder: {
+    deleteConfirmation:
+      'Nästlade mappar kommer att raderas. Nästlade dokument kommer att flyttas till den närmaste föräldramappen. Är du säker?',
```

Comment:
If you have time to replace this rather weirdly sounding phrase with the one below it would be nice!

'Eventuella underliggande mappar kommer också tas bort. Filerna i de mapparna flyttas upp till samma plats som mappen du tar bort. Är du säker?'